Doc enhancements based on AR feedback (#97)
Esri/arcgis-runtime-toolkit-ios: Documentation/README.md

### Table of Contents

* [Augmented reality (AR)](AR/readme) - Integrates the scene view with ARKit to enable augmented reality (AR).
* [Bookmarks](Bookmarks/readme) - Shows bookmarks, from a map, scene, or a list.
* [Compass](Compass/readme) - Shows a compass direction when the map is rotated. Auto-hides when the map points north up.
* [JobManager](JobManager/readme) - Suspends and resumes ArcGIS Runtime tasks when the app is background, terminated, and relaunched.

> * [LegendViewController](LegendViewController/readme) - Displays a legend for a single layer in your map (and optionally for its sub layers).

Reviewer: ```suggestion
* [LegendViewController](LegendViewController/readme) - Displays a legend for all the layers in a map or scene contained in an `AGSGeoView`.
```
